compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion/Primitives.hs

```diff
@@ -199,7 +199,7 @@ _whenRuntimeSupports version feature t e
     | otherwise = runtimeUnsupported feature t
 
 runtimeUnsupported :: Feature -> Type -> Expr
-runtimeUnsupported (Feature name version) t =
+runtimeUnsupported (Feature name version _) t =
```

Comment:
We should be able to remove this function now. This was our old hack around the fact that the stdlib could not depend on the DAML-LF version we're compiling to.